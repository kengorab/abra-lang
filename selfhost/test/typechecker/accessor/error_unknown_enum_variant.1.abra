enum Foo { Bar }

val f = Foo.Baz
