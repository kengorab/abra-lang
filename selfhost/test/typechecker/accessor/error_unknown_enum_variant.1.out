Error at %FILE_NAME%:3:13
Unknown field 'Baz'
  |  val f = Foo.Baz
                 ^
No field 'Baz' found on type <#enum Foo>
